### DISCUSSION TITLE
bug：error: OpenCV(4.9.0) /io/opencv/modules/core/src/arithm.cpp:647: error: (-209:Sizes of input arguments do not match) The operation is neither 'array op array' (where arrays have the same size  and the same number of channels), nor 'array op scalar', nor 'scalar op array' in function 'arithm_op'

### DISCUSSION THREAD

**Discussion by zhaozzt** [open-edge-platform/anomalib#1857] [Q&A]
![6](https://github.com/openvinotoolkit/anomalib/assets/127210381/8506d6a4-520b-4bf3-9c6b-5dba53f52586)
![Uploading 7.png…]()

**Comment by xuzhengzhenghit** [comment]
哥们你配置文件发下看看

**Reply by zhaozzt** [reply]
这和配置文件有关系吗？

**Reply by xuzhengzhenghit** [reply]
或者是你的那个mask有问题？那个好像是灰度图？

**Reply by zhaozzt** [reply]
![8](https://github.com/openvinotoolkit/anomalib/assets/127210381/c8f3517f-f3d4-439c-8197-2df1553cd429)
那mask图我用0.7就能用，为什么1.0就出这个错误
